### PR TITLE
update chagnelog for kube-1.24

### PR DESCRIPTION
**Pull request by kschoche**
Changes proposed in this PR:
- adds note about required changes for multiport users.

How I've tested this PR:

How I expect reviewers to test this PR:

Checklist:
- [ ] Tests added
- [x] CHANGELOG entry added 
  > HashiCorp engineers only, community PRs should not add a changelog entry.
  > Entries should use present tense (e.g. Add support for...)

### CODE REVIEW

**Review by david-yu — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,26 @@ FEATURES:
 * Kubernetes 1.24
   * Add support for Kubernetes 1.24 where ServiceAccounts no longer have long term JWT tokens. [[GH-1431](https://github.com/hashicorp/consul-k8s/pull/1431)]
   * Upgrade kubeVersion in helm chart to support Kubernetes 1.21+.
+  NOTE: Users deploying multiple services to the same Pod (multiport) must also deploy an Kube Secret for each ServiceAccount associated with the Consul service. The name of the Secret must match the ServiceAccount name and be of type 'kubernetes.io/service-account-token'
```
Comment (lkysow) [inline CHANGELOG.md:18]: * looks like there's two FEATURES sections?
* Can you put this under a BREAKING CHANGES section. Is this only needed if you're using Kube 1.24+? We should note that if true.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,26 @@ FEATURES:
...
+metadata:
+  name: svc1
+  annotations:
+    kubernetes.io/service-account.name: svc1
+type: kubernetes.io/service-account-token
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: .multiport
```
Comment (lkysow) [inline CHANGELOG.md:32]: nit: secret names can't have `.`'s can they?

**Review by lkysow — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (kschoche) [reply CHANGELOG.md:32]: Oof good catch this was a poor copy paste lol

File: CHANGELOG.md (hunk 1 above)
Comment (kschoche) [reply CHANGELOG.md]: Updated it in the breaking changes section.

**Pull request merged**